cmd/kcp-catalog/main.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	goflags "flag"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/component-base/cli"
+	"k8s.io/component-base/version"
+	"k8s.io/klog/v2"
+
+	"github.com/kcp-dev/kcp/pkg/cmd/help"
+)
+
+func main() {
+	cmd := &cobra.Command{
+		Use:   "kubectl-kcp",
+		Short: "kubectl-kcp",
+		Long: help.Doc(`
+			kcp is a CLI tool to manage Catalog API objects.
+		`),
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+
```

Comment:
Missing adding the bind command?

Reply:
Its here (https://github.com/kcp-dev/catalog/pull/6/files#diff-67a3933cefad93d0eac14140de732c2812c0c3787f51d4134edf5ce2274219a5R62), my bad changes weren't up to date.